vJoySerialFeeder/SerialProtocols/CrsfReader.cs

```diff
@@ -126,6 +126,7 @@ public override int ReadChannels()
 				} /* if packed channels */
 			} /* if valid checksum */
 
+			Buffer.FrameLength = 2;
```

Comment:
Now when I look at the code I think it'll be cleaner if we put this in front of
```c#
Buffer.FrameLength = 2;
byte len = Buffer[1];
//byte addr = Buffer[0];
```
and remove it from `Start()`

In this way it'll be cleaner. We set the length to `2` and then read `len`.
`addr` is not strictly needed. I mean you can leave it commented, together with the debug line, but effectively it is not needed. `Buffer[1]` will consume the first (0th) byte anyway.
If `len` is valid we set the proper `FrameLength` and continue further.